website/blog/2023-11-11-AgentEval/index.mdx

```diff
@@ -0,0 +1,144 @@
+
+---
+title:  AgentEval: Assessing Task Utility for LLM-powered Applications
+authors: julianakiseleva, narabzad
+tags: [LLM, GPT, evaluation, task utility]
+---
+
+![Fig.1: An verification framework](img/agenteval-CQ.png)
+
+**TL;DR:**
+* We introduce a notion of what is task utility in LLM-powered applications
+* We introduce `AgentEval` — the first version of the framework to assess task utility for an arbitrary application. It automaticaly suggests criteria to explain task utility and then quantifies these criteria for logs of your application.

```

Comment:
We introduce `AgentEval` — the first version of an evaluation framework to assess task utility for an arbitrary application. It automatically suggests criteria to explain and describe task utility and then quantifies these criteria for logs of your application.

Reply:
@nushib I rewrote the TL;DR: part completely. please check it after PR will be updated.